index.html

```diff
@@ -47,5 +47,332 @@ <h1 class="headline-1-h1">I’m Mariane
             </ul>
         </div>
     </section>
+    <!--Works Section-->
+    <section class="works">
+        <!--works1-->
+        <section class="works-1">
+                <img class="works-1-div1-img" src="images/Snapshoot Portfolio1.svg" alt="">
+            <div class="works-1-div2">
+                <div class="works-1-div2-1">
+                    <div class="works-1-div2-1-a">
+                        <p class="works-1-div2-1-a-p">Tonic</p>
```

Comment:
- [ ] to be semantically correct , the card titles example : 'Tonic' should use `<hx> (h2,h3,etc...)` tag , kindly fix it for all of your cards 🙏